<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1007</width>
    <height>932</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="main_widget">
   <widget class="QFrame" name="palm_frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>995</width>
      <height>916</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	border: 1px solid rgba(0,0,0, .12);
	border-radius: 7px;
	background-color: white;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pb_save">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>816</x>
       <y>13</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Tw Cen MT Condensed</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: #DCFBED;
	color: #2EA76F;
	border: 1px solid white;
	border-radius: 10px;
	padding-bottom: 1px;
}
QPushButton:pressed {
	margin: 1 1 1 1;
}</string>
     </property>
     <property name="text">
      <string>SAVE CSV</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="view_canvas">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>46</y>
       <width>962</width>
       <height>850</height>
      </rect>
     </property>
     <property name="cursor" stdset="0">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #EFF8F4;
border: none;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="transformationAnchor">
      <enum>QGraphicsView::AnchorUnderMouse</enum>
     </property>
     <property name="resizeAnchor">
      <enum>QGraphicsView::AnchorUnderMouse</enum>
     </property>
    </widget>
    <widget class="QLabel" name="info_display">
     <property name="geometry">
      <rect>
       <x>688</x>
       <y>13</y>
       <width>121</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Tw Cen MT</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #EFF8F4;
border: 1px solid white;
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_prob">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>899</x>
       <y>13</y>
       <width>78</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Tw Cen MT Condensed</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: #DCFBED;
	color: #2EA76F;
	border: 1px solid white;
	border-radius: 10px;
	padding-bottom: 1px;
}
QPushButton:pressed {
	margin: 1 1 1 1;
}</string>
     </property>
     <property name="text">
      <string>PROB MAP</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_crop_size">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>475</x>
       <y>13</y>
       <width>100</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Tw Cen MT</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
	border: 1px solid white;
	border-radius: 10px;
	background-color: #F1F1F1;
	padding-bottom: 1px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_overlap_ratio">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>582</x>
       <y>13</y>
       <width>100</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Tw Cen MT</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
	border: 1px solid white;
	border-radius: 10px;
	background-color: #F1F1F1;
	padding-bottom: 1px;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>6</y>
       <width>141</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="pb_leave">
      <property name="geometry">
       <rect>
        <x>4</x>
        <y>4</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tw Cen MT Condensed</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: rgba(0,0,0,0);
	border: none;
}
QPushButton:pressed{
	margin: 1 1 1 1;
	outline: 0;
}


</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../GUIImg/open-exit-door.png</normaloff>../GUIImg/open-exit-door.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_openfile">
      <property name="geometry">
       <rect>
        <x>31</x>
        <y>4</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tw Cen MT Condensed</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: rgba(0,0,0,0);
	border: none;
}
QPushButton:pressed{
	margin: 1 1 1 1;
	outline: 0;
}


</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../GUIImg/computer-folder2.png</normaloff>../GUIImg/computer-folder2.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_loadcsv">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>59</x>
        <y>4</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tw Cen MT Condensed</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: rgba(0,0,0,0);
	border: none;
}
QPushButton:pressed{
	margin: 1 1 1 1;
	outline: 0;
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../GUIImg/geo.png</normaloff>../GUIImg/geo.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>19</width>
        <height>19</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_dataset">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>84</x>
        <y>4</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tw Cen MT Condensed</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: rgba(0,0,0,0);
	border: none;
}
QPushButton:pressed{
	margin: 1 1 1 1;
	outline: 0;
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../GUIImg/direct-download.png</normaloff>../GUIImg/direct-download.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
